[2025-01-22T10:33:58.119+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-22T10:33:58.130+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: branching_variables.branch manual__2025-01-22T10:33:54.927465+00:00 [queued]>
[2025-01-22T10:33:58.136+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: branching_variables.branch manual__2025-01-22T10:33:54.927465+00:00 [queued]>
[2025-01-22T10:33:58.138+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-22T10:33:58.147+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): branch> on 2025-01-22 10:33:54.927465+00:00
[2025-01-22T10:33:58.153+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1831) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-22T10:33:58.155+0000] {standard_task_runner.py:72} INFO - Started process 1833 to run task
[2025-01-22T10:33:58.155+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'branching_variables', 'branch', 'manual__2025-01-22T10:33:54.927465+00:00', '--job-id', '228', '--raw', '--subdir', 'DAGS_FOLDER/branching_variables.py', '--cfg-path', '/tmp/tmpk5mtuyxc']
[2025-01-22T10:33:58.157+0000] {standard_task_runner.py:105} INFO - Job 228: Subtask branch
[2025-01-22T10:33:58.186+0000] {task_command.py:467} INFO - Running <TaskInstance: branching_variables.branch manual__2025-01-22T10:33:54.927465+00:00 [running]> on host 7854821b9569
[2025-01-22T10:33:58.330+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohamed' AIRFLOW_CTX_DAG_ID='branching_variables' AIRFLOW_CTX_TASK_ID='branch' AIRFLOW_CTX_EXECUTION_DATE='2025-01-22T10:33:54.927465+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-22T10:33:54.927465+00:00'
[2025-01-22T10:33:58.331+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-22T10:33:58.338+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-01-22T10:33:58.343+0000] {python.py:240} INFO - Done. Returned value was: filter_southwest
[2025-01-22T10:33:58.343+0000] {branch.py:38} INFO - Branch into filter_southwest
[2025-01-22T10:33:58.344+0000] {skipmixin.py:230} INFO - Following branch ('filter_southwest',)
[2025-01-22T10:33:58.348+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-01-22T10:33:58.365+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-22T10:33:58.366+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=branching_variables, task_id=branch, run_id=manual__2025-01-22T10:33:54.927465+00:00, execution_date=20250122T103354, start_date=20250122T103358, end_date=20250122T103358
[2025-01-22T10:33:58.411+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-22T10:33:58.420+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-22T10:33:58.423+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
